src/components/editor/ResourceTemplate.jsx

```diff
@@ -32,7 +32,7 @@ class ResourceTemplate extends Component {
       <div className="ResourceTemplate">
         <div id="resourceTemplate" style={{ marginTop: '-30px' }}>
           <section className="col-md-9">
-            <h1><em>{this.props.title}</em></h1>
+            <h1><em>{this.props.rtLabel}</em></h1>
```

Comment:
again: it's a label in the context of the resource template. It's a title in the context of the html.  We know it's a title here because of the h1.